packages/frontend/amp/components/elements/InteractiveMarkup.tsx

```diff
@@ -24,6 +24,25 @@ export const InteractiveMarkup: React.SFC<{
     const styleTag = styles ? `<style>${styles}</style>` : '';
     const scripts = js ? `<script>${js}</script>` : '';
 
+    const resizeScript = `
+<script>
+    var height = Math.max(
+        body.scrollHeight,
+        body.offsetHeight, 
+        html.clientHeight,
+        html.scrollHeight,
+        html.offsetHeight
+    );
+    
+    window.parent.postMessage({
+        sentinel: 'amp',
+        type: 'embed-size',
+        height: height
+    }, '*');
+    
+    console.log(height);
```

Comment:
Did this script sneak in? Looks cool, just checking... (though console.logggy maybe shouldn't be here)

Reply:
Yeah this is a mistake from a previous PR. I'll clean it up!

Reply:
And actually I'm not even sure if this works yet so going to pull it for now.

Reply:
Oh. Thanks @gtrufitt !